src/test/resources/features/createContract.feature

```diff
@@ -27,3 +27,158 @@ Feature: Create Contract
       | Description             | Description         |
     Then I verify Contract created with all fields
       And I verify that the date matches the creation date
+
+  @CreateContract
+  Scenario: create a contract with Test Account as pre-postcondition
```

Comment:
Sentences should start in capital letter.

Reply:
Ok, It was solved